static/js/app.js

```diff
@@ -129,128 +129,106 @@ var SUBTYPE_STRING = [
     "פגיעה ממטען של רכב"
     ];
 
-//agamadd- infowindow to use as tour
+//agam add - var for infowindow,checkStepsAdd,tourLocation to use as tour
 var infowindow;
-function step2next(){
-    infowindow.close();
-    tour1.init();
-    tour1.start();
-}
-function step2prev(){
-    infowindow.close();
-    tour.prev();
-}
-
-//agam add- add default val to search input
-var tourLocation =0;
-function tourAddInput(){
-    var a = document.getElementById('pac-input');
-        a.value="זבוטנסקי 74, פתח תקווה";
-    }
-function tourClickInput(){
-    var a = document.getElementById('pac-input');
-        a.value="זבוטנסקי 74, פתח תקווה";
-        google.maps.event.trigger( a, 'focus');
-        google.maps.event.trigger( a, 'keydown', {keyCode:13});
-        tourLocation = 2;
-    }
-$(function() {
-    //agam add- tour button On Click trigger
-    $('#tourOnClick').on('click', function () {
-        //clear cache
-        tour.restart()
-        // Start the tour
-        tour.start();
-    });
-    var AppRouter = Backbone.Router.extend({
-        routes: {
-            "" : "navigateEmpty",
-            "/?marker=:id&start_date=:start&end_date=:end&show_fatal=:showFatal&show_severe=:showSevere&show_light=:showLight&show_inaccurate=:showInaccurate&zoom=:zoom&lat=:lat&lon=:lon" : "navigate"
-        },
-        navigate: function(id, start, end, showFatal, showSevere, showLight, showInaccurate, zoom, lat, lon) {
-            app.model.set("currentMarker", parseInt(id));
-            app.model.set("dateRange", [new Date(start), new Date(end)]);
-            app.model.set("showFatal", showFatal);
-            app.model.set("showSevere", showSevere);
-            app.model.set("showLight", showLight);
-            app.model.set("showInaccurateMarkers", showInaccurate != 0);
-            app.map.setZoom(zoom);
-            app.map.setCenter(new google.maps.LatLng(lat, lon));
-        },
-        navigateEmpty: function() {
-            app.model.set("currentMarker", null);
-        }
-    });
+var checkStepsAdd = false;
+var tourLocation = 0;
+
+    $(function () {
```

Comment:
later after the game...

2015-05-06 21:29 GMT+03:00 אגם אלוני <agamaloni@gmail.com>:

> yes  :) thanks for the help.
>
> 2015-05-06 21:24 GMT+03:00 Daniel Hershcovich <notifications@github.com>:
>
>> In static/js/app.js
>> <https://github.com/hasadna/anyway/pull/241#discussion_r29786830>:
>>
>> > -            app.model.set("dateRange", [new Date(start), new Date(end)]);
>> > -            app.model.set("showFatal", showFatal);
>> > -            app.model.set("showSevere", showSevere);
>> > -            app.model.set("showLight", showLight);
>> > -            app.model.set("showInaccurateMarkers", showInaccurate != 0);
>> > -            app.map.setZoom(zoom);
>> > -            app.map.setCenter(new google.maps.LatLng(lat, lon));
>> > -        },
>> > -        navigateEmpty: function() {
>> > -            app.model.set("currentMarker", null);
>> > -        }
>> > -    });
>> > +var checkStepsAdd = false;
>> > +var tourLocation = 0;
>> > +
>> > +    $(function () {
>>
>> So are you updating this pull request?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/hasadna/anyway/pull/241/files#r29786830>.
>>
>
>